cmd/fluxd/main.go

```diff
@@ -134,6 +133,8 @@ func main() {
 			logger.Log("err", err)
 			os.Exit(1)
 		}
+		clientset.Core()
```

Comment:
What's this do? Some side-effect, presumably. Please document in a comment, or remove.